File: app/src/main/java/com/akrivonos/pi_school_w2/Api/ApiRetrofitInterface.java
package com.akrivonos.pi_school_w2.Api;

import com.akrivonos.pi_school_w2.Models.Rsp;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface ApiRetrofitInterface {
    @GET("?method=flickr.photos.search")
    Call<Rsp> searchPhotosByName(@Query("api_key") String api_key, @Query("text") String searchText);
}

File: app_kotlin/src/main/java/com/akrivonos/app_kotlin/Api/ApiRetrofitInterface.kt
package com.akrivonos.app_kotlin.Api

import com.akrivonos.app_kotlin.Models.Rsp

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface ApiRetrofitInterface {
    @GET("?method=flickr.photos.search")
    fun searchPhotosByName(@Query("api_key") api_key: String, @Query("text") searchText: String): Call<Rsp>
}
